xbmc/music/Album.cpp

```diff
@@ -135,14 +138,19 @@ CAlbum::CAlbum(const CFileItem& item)
         musicBrainzAlbumArtistHints = tag.GetAlbumArtist();
     }
 
+    // Try to get number of artist sort names and musicbrainz ids to match. Split sort names 
+    // further using multiple possible delimiters, over single separator applied in Tag loader
+    if (artistSort.size() != tag.GetMusicBrainzAlbumArtistID().size())
+      artistSort = StringUtils::SplitMulti(artistSort, { ";", ":", "|", "#" });
+
     for (size_t i = 0; i < tag.GetMusicBrainzAlbumArtistID().size(); i++)
```

Comment:
oh didn't notice that. I caught it in another place where I was starting to comment and cancelled it. Yeah ignore my comments about the for loops.